backend/PIMTool.Core/Helpers/ExcelStyleHelper.cs

```diff
@@ -0,0 +1,44 @@
+﻿using NPOI.HSSF.UserModel;
+using NPOI.SS.UserModel;
+using NPOI.XSSF.UserModel;
+
+namespace PIMTool.Core.Helpers;
+
+public static class ExcelStyleHelper
+{
+    public static ICellStyle SetBackgroundColor(this IWorkbook workbook, IndexedColors color)
```

Comment:
This is a feature that I just want to try out (for learning purpose). I think it might be helpful for my other projects in the future. It won't affect the UI of the origin project though.